i18n/en/design-system.json

```diff
@@ -6,13 +6,13 @@
   "global-footer-fandom-follow-us-alt-youtube": "Youtube",
   "global-footer-fandom-follow-us-header": "Follow Us",
   "global-footer-fandom-link-fan-communities": "Fan Communities",
-  "global-footer-fandom-link-vertical-games": "Games",
-  "global-footer-fandom-link-vertical-movies": "Movies",
-  "global-footer-fandom-link-vertical-tv": "TV",
-  "global-footer-fandom-link-vertical-books": "Books",
-  "global-footer-fandom-link-vertical-comics": "Comics",
-  "global-footer-fandom-link-vertical-lifestyle": "Lifestyle",
-  "global-footer-fandom-link-vertical-music": "Music",
+  "global-footer-vertical-games": "Games",
```

Comment:
I don't think this is a good idea. Right until now we had 1:1 mapping API fields -> message keys.
Also, links in Fandom section are in different context from those in the bottom bar.
I'd revert this one and name the new one `global-footer-licensing-and-vertical-description-param-vertical-games`